# Array of frontier clusters
std_msgs/Header header
frontier_exploration/FrontierCluster[] clusters